<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowExtractData</class>
 <widget class="QWidget" name="WindowExtractData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>429</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Grande</family>
    <pointsize>20</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="labelTitleExtract">
     <property name="minimumSize">
      <size>
       <width>123</width>
       <height>114</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Extract data</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QToolButton" name="settingsButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="buttonExtract">
     <property name="text">
      <string>Start the extraction</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="labelExtractWait">
     <property name="text">
      <string>Please wait while the program is extracting the datas.</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QListWidget" name="listWidgetId"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>100</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QCheckBox" name="checkBoxSuccesExtract">
     <property name="text">
      <string>Succes Extract</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="NextExtract">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
